### PR TITLE
[ * Interview Question?]

### PR DESCRIPTION
**Pull request by johngorithm**
#### What this PR does?
- Add question on data-flow in stateful widgets
- Add question on `InheritedWidget`s
- Add question on `FAB` usage
- Add question on Dart `Future`

#### Note
The aim is to help learner prepare for questions around data-flow in stateful widgets, `InheritedWidget`s and `Future`s

### CODE REVIEW

File: Source.md [hunk 1]
```diff
@@ -26,3 +26,8 @@ title: Flutter Interview Questions
 * What is the role of a Change Notifier Provider?
 * What is a ValueListenableBuilder?
 * What is an AspectRatio?
+* How would you access Stateful Widget properties from it's State?
+* What is InheritedWidget in Flutter?
+* Is there a suggested limit to the number of FloatingActionButtons a screen can have?. Give reason(s) for your answer
+* What is a Future in Dart?
```
Comment (lucalves) [inline Source.md]: I think this specific question is repeated in line 41:

> Explain async, await and Future?

Comment (johngorithm) [reply Source.md]: Good catch @lucalves. Will get rid of that

Comment (lucalves) [reply Source.md]: I think it's perfect now, @johngorithm.

**Comment by whatsupcoders** [comment]
Thanks @johngorithm for your contribution.

**Pull request merged**